### PR TITLE
Adding cucushift

### PR DESCRIPTION
**Pull request by jhou1**
The image built from cucushift repo will be used as the base image for this one, I tested this locally by replacing the `FROM` with cucushift image. 

@liangxia PTAL, thank you!

### CODE REVIEW

File: tools/openshift-ci/Dockerfile [hunk 1]
```diff
@@ -3,34 +3,26 @@ FROM quay.io/centos/centos:7
...
+ADD . /verification-tests/
+
+RUN set -x && \
+    mv /tierN/ /verification-tests/tierN/
+
 RUN set -x && \
     yum -y update && \
     SCL_BASE_PKGS="centos-release-scl scl-utils-build" && \
-    INSTALL_PKGS="rh-ruby27 rh-ruby27-ruby-devel bsdtar git openssh-clients" && \
+    INSTALL_PKGS="rh-ruby26 rh-ruby26-ruby-devel rh-git218 bsdtar" && \
```
Comment (liangxia) [inline tools/openshift-ci/Dockerfile:16]: Perhaps use ruby27 ? as the image we currently use is using ruby27.

**Comment by liangxia** [comment]
/lgtm

/hold
in case you do not want to merge it now.

File: tools/openshift-ci/Dockerfile (hunk 1 above)
Comment (jhou1) [reply tools/openshift-ci/Dockerfile]: Thanks Liang, fixed that

**Comment by jhou1** [comment]
Thanks, Liang, it's fine to unhold. Current openshift/release is building the Dockerfile.centos

**Pull request merged**